pkg/helm/helm_test.go

```diff
@@ -115,18 +118,29 @@ func TestHelmRelease(t *testing.T) {
 			"key": "value1",
 		},
 	}
-	_, err = impl.Install(origChart, kubeNamespace, releaseName, vals)
+	_, err = impl.Install(origChart, kubeNamespace, releaseName, vals, true)
 	assert.Nil(t, err)
 	Log(t, "install", err)
 
-	_, err = impl.Upgrade(origChart, kubeNamespace, releaseName, vals)
+	_, err = impl.Upgrade(origChart, kubeNamespace, releaseName, vals, true)
 	assert.Nil(t, err)
 	Log(t, "upgrade", err)
 
-	_, err = impl.Status(kubeNamespace, releaseName)
-	assert.Nil(t, err)
+	assert.Eventually(t, func() bool {
```

Comment:
Seems to check status update rather than get resources, is it necessary?

Reply:
Yes. "wait" is true, and we wait for the release to be deployed before checking the resources.